firmware/src/core/screenmanager/ScreenManager.h

```diff
@@ -78,12 +80,19 @@ class ScreenManager {
     void drawLegacyString(const String &string, int32_t x, int32_t y, uint8_t font);
     int16_t drawLegacyChar(uint16_t uniCode, int32_t x, int32_t y, uint8_t font);
 
+    // Image functions
+    static bool tftOutput(int16_t x, int16_t y, uint16_t w, uint16_t h, uint16_t *bitmap);
+    JRESULT drawJpg(int32_t x, int32_t y, const uint8_t jpeg_data[], uint32_t data_size, uint8_t scale = 1, uint32_t imageColor = 0);
```

Comment:
Yes, that would be one option to approach this, but that's beyond the scope of this PR.

I think for now, it makes sense to consolidate all drawing functions in one class. This will also make it easier to refactor it later.

At the moment, the ScreenManager class is responsible for all screen manipulations (i.e. select a screen and draw to it).
It would also be easily usable for single screen "Mini-Widgets" by passing the screen number to SingleScreenWidget.draw() (or even preselecting the correct screen before calling draw()).
Like it's already the case for most widgets (e.g. `StockWidget::displayStock(int8_t displayIndex, ...)`, `WeatherWidget::threeDayWeather(int displayIndex)`).

IMHO the bigger challenge is separating the data/update functions from the draw functions so that we can use a single update() to get the data but use it in multiple SingleScreenWidgets. For that we probably need some kind of DataManager class where the SingleScreenWidgets can register their data needs and the DataManager is responsible for updating the data as necessary.